README.md

```diff
@@ -68,7 +68,8 @@ We would appreciate any external contributions! 🙏
     * If you're building your tasks based existing datasets and their crowdsourcing templates, see these [guidelines](doc/crowdsourcing.md). 
  * Add your task to [our list of tasks](tasks/README.md).
  * To make sure that your addition is formatted correctly, run the tests: `> python src/test_all.py`
-  
+ * To automatically update README after modifying "Categories", "Domains", "Input_language" or "Output_language", run : `> python src/auto_update_readme.py`  
```

Comment:
I would prefer if we drop this. Most users don't need to worry about this.

Reply:
Done!